app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt

```diff
@@ -83,13 +239,27 @@ class OnlineLibraryFragment : LibraryFragment(), FragmentActivityExtensions {
     setHasOptionsMenu(true)
     val root = inflater.inflate(R.layout.fragment_destination_download, container, false)
     val toolbar = root.findViewById<Toolbar>(R.id.toolbar)
+    val swipeRefreshLibrary = root.findViewById<SwipeRefreshLayout>(R.id.librarySwipeRefresh)
```

Comment:
there should be no need for these new findViewByIds

Reply:
Going with KAE resulted in npe

Reply:
Because this is inside `onCreateView`,  the view isn't "created" until the inflated resource is returned from this method

override `OnViewCreated` and KAE will work